<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReloginForm</class>
 <widget class="QDialog" name="ReloginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Please enter your password</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../huggle_res/pictures.qrc">
    <normaloff>:/huggle/pictures/Resources/huggle3_newlogo.ico</normaloff>:/huggle/pictures/Resources/huggle3_newlogo.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>It looks like you were logged out of MediaWiki for unknown reason. You can either try to login again, or you can terminate the program without saving your on-wiki configuration. Please enter your password to login again:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="inputMethodHints">
      <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_RemeberPw">
     <property name="text">
      <string>remember</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>[[autorelog]]</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_ForceExit">
       <property name="text">
        <string>Exit huggle without saving your configs</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Relog">
       <property name="text">
        <string>Relog</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../huggle_res/pictures.qrc"/>
 </resources>
 <connections/>
</ui>
